run-build-for-gh-deps.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# TODO: This file is temporary and will be removed after testing
```

Comment:
Did you try [this approach](https://github.com/openedx/frontend-build#local-module-configuration-for-webpack) instead? Should be much simpler than a whole custom script.

Reply:
Hmm, I don't think this will work for sandbox deployment. In the module config we specify the path to local folders that are not present during deployment

Reply:
OK. Please remember to remove this file before merging.

Reply:
Please go ahead and remove this workaround.  I'm reviewing and testing this locally, and things are working well enough that we're almost ready to merge the frontend-component-header change, anyway.